grid-proxy/tests/queries/farm_test.go

```diff
@@ -41,6 +41,11 @@ var farmFilterRandomValueGenerator = map[string]func(agg FarmsAggregate) interfa
 		name := changeCase(agg.farmNames[rand.Intn(len(agg.farmNames))])
 		return &name
 	},
+	"Country": func(_ FarmsAggregate) interface{} {
```

Comment:
why not like other fields with FarmsAggregate

Reply:
the `agg` variable is not intended to be used inside this function. however, it is still needed to satisfy the function signature. I set it as blank to bypass the warnings

Reply:
the farm table itself doesn't have a country field so we got the possible counties from the node aggregation